tomee/tomee-microprofile/mp-common/pom.xml

```diff
@@ -84,7 +206,7 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>mp-jwt</artifactId>
-      <version>${microprofile.jwt.impl.version}</version>
+      <version>${project.version}</version>
```

Comment:
See previous comment. If you don't change the property, it has ${project.version} already.

Reply:
rolledback this change.